google/cloud/storage/examples/storage_notification_samples.cc

```diff
@@ -94,8 +107,15 @@ void GetNotification(google::cloud::storage::Client client, int& argc,
   //! [get notification] [START storage_print_pubsub_bucket_notification]
   namespace gcs = google::cloud::storage;
   [](gcs::Client client, std::string bucket_name, std::string notification_id) {
-    gcs::NotificationMetadata notification =
+    gcs::StatusOr<gcs::NotificationMetadata> status =
         client.GetNotification(bucket_name, notification_id);
+    if (not status.ok()) {
+      std::cerr << "Error getting notification metadata for notification id "
+                << notification_id << " on bucket " << bucket_name
+                << ", status=" << status.status() << std::endl;
+      return;
+    }
+    gcs::NotificationMetadata notification = std::move(*status);
```

Comment:
Being explicit here would also be helpful.

`std::move(status.value());`

I'm open to interpretation, but it would help to not guess what this is doing.

Reply:
See above about `operator*` vs. `.value()`.

Reply:
FWIW, I made a related comment above, but in a different direction. Just mentioning here because I didn't see this comment originally.